xml/decoders/0NoDecoder.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<?xml-stylesheet type="text/xsl" href="../XSLT/decoder.xsl"?>
+<!-- Copyright (C) JMRI 2024 All rights reserved -->
+<!--                                                                        -->
+<!-- JMRI is free software; you can redistribute it and/or modify it under  -->
+<!-- the terms of version 2 of the GNU General Public License as published  -->
+<!-- by the Free Software Foundation. See the "COPYING" file for a copy     -->
+<!-- of this license.                                                       -->
+<!--                                                                        -->
+<!-- JMRI is distributed in the hope that it will be useful, but WITHOUT    -->
+<!-- ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or  -->
+<!-- FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License  -->
+<!-- for more details.                                                      -->
+<decoder-config xmlns:xi="http://www.w3.org/2001/XInclude" 
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+    xsi:noNamespaceSchemaLocation="http://jmri.org/xml/schema/decoder-4-15-2.xsd"
+    showEmptyPanes="no">
+  <copyright xmlns="http://docbook.org/ns/docbook">
+    <year>2024</year>
+    <holder>JMRI</holder>
+  </copyright>
+  <authorgroup xmlns="http://docbook.org/ns/docbook">
+    <author>
+      <personname><firstname>Bob</firstname><surname>Jacobsen</surname></personname>
+    </author>
+  </authorgroup>
+  <revhistory xmlns="http://docbook.org/ns/docbook">
+    <revision>
+      <revnumber>0</revnumber><date>2024-06-08</date><authorinitials>BJ</authorinitials>
+      <revremark>Initial version</revremark>
+    </revision>
+  </revhistory>
+  <decoder>
+    <family name="DC" mfg="No Decoder">
+      <model model="DC"/>
+    </family>
+    <programming direct="no" paged="no" register="no" ops="no"/>
```

Comment:
No need to change, but a question.

When I try the PR and click "New loco" and then "Open Comprehensive Programmer", I get the error:
`Cannot set mode`.

Would it be difficult to add a new `nodecoder` mode and make the buttons `Read ...`, `Write ...` and `Compare ...` disabled in this mode? It would get rid of the error and I think it would be a good safety precaution since a loco with "No decoder" should never be programmed.

Reply:
> Would it be difficult to add a new nodecoder mode and make the buttons Read ..., Write ... and Compare ... disabled in this mode? It would get rid of the error and I think it would be a good safety precaution since a loco with "No decoder" should never be programmed.

The idea of programming a decoder is buried pretty deeply in that code.  Everything is possible, but I don't think I can get that done before the production release.

Reply:
> The idea of programming a decoder is buried pretty deeply in that code. Everything is possible, but I don't think I can get that done before the production release.

PR #13182 disables the buttons if the roster entry has no decoder. It can wait for 5.9.1.